client/__tests__/e2e/e2e.test.ts

```diff
@@ -101,6 +104,143 @@ const testURLs = {
   "super-cool-spatial.cxg": pageURLSpatial,
 };
 
+describe(`Testing CellGuideCXG at ${pageURLCellGuide}`, () => {
+  test("page launched", async ({ page }, testInfo) => {
+    await goToPage(page, pageURLCellGuide);
+
+    const element = await page.getByTestId("header").innerHTML();
+
+    expect(element).toMatchSnapshot();
+
+    await snapshotTestGraph(page, testInfo);
+  });
+  test("assert absence of 'Standard Categories' and 'Author Categories'", async ({
+    page,
+  }) => {
+    await goToPage(page, pageURLCellGuide);
+
+    // Check for the absence of 'Standard Categories'
+    const standardCategories = await page
+      .locator("text='Standard Categories'")
+      .count();
+    expect(standardCategories).toBe(0);
+
+    // Check for the absence of 'Author Categories'
+    const authorCategories = await page
+      .locator("text='Author Categories'")
+      .count();
+    expect(authorCategories).toBe(0);
+  });
+  test("assert 'Marker Gene Sets' header is present and collapsed", async ({
+    page,
+  }) => {
+    await goToPage(page, pageURLCellGuide);
+
+    // Check for the presence of 'Marker Gene Sets' header
+    const markerGeneSetsHeader = await page.locator(
+      "h5:has-text('Marker Gene Sets')"
+    );
+    await expect(markerGeneSetsHeader).toBeVisible();
+
+    // Check if the header is collapsed by looking for a child with a chevron-right icon
+    const chevronRightIcon = markerGeneSetsHeader.locator(
+      "svg[data-icon='chevron-right']"
+    );
+    await expect(chevronRightIcon).toBeVisible();
+  });
+  test("expansion of 'Marker Gene Sets' reveals specific genesets", async ({
+    page,
+  }) => {
+    await goToPage(page, pageURLCellGuide);
+
+    // Locate and expand the 'Marker Gene Sets' header
+    const markerGeneSetsHeader = await page.locator(
+      "h5:has-text('Marker Gene Sets')"
+    );
+    await markerGeneSetsHeader.click(); // Assuming clicking will expand the section
+
+    tryUntil(
+      async () => {
+        // Assert the presence of specific genesets
+        const geneset1 = await page.locator(
+          `div[data-testid="geneset"]:has-text("enteric smooth muscle cell")`
+        );
+        const geneset2 = await page.locator(
+          `div[data-testid="geneset"]:has-text("smooth muscle fiber of ileum")`
+        );
+
+        await expect(geneset1).toBeVisible();
+        await expect(geneset2).toBeVisible();
+      },
+      { page }
+    );
```

Comment:
discussion: I'm thinking the click should be part of the tryUntil, in case the click is not registered and the section is never expanded, then the tryUntil will never succeed

Reply:
I moved the click outside because I thought it would be bad to toggle it open/closed each iteration of tryUntil, but since it runs a bunch of times I guess i would be fine.